opengrok-indexer/src/main/java/org/opengrok/indexer/web/Scripts.java

```diff
@@ -103,11 +109,15 @@ public String toHtml() {
         SCRIPTS.put("jquery", new FileScript("js/jquery-3.4.1.min.js", 10));
         SCRIPTS.put("jquery-ui", new FileScript("js/jquery-ui-1.12.1-custom.min.js", 11));
         SCRIPTS.put("jquery-tablesorter", new FileScript("js/jquery-tablesorter-2.26.6.min.js", 12));
-        SCRIPTS.put("tablesorter-parsers", new FileScript("js/tablesorter-parsers-0.0.1.js", 13));
+        SCRIPTS.put("tablesorter-parsers", new FileScript("js/tablesorter-parsers-0.0.1-min.js", 13));
```

Comment:
i've seen the general consent about the nameing is the filename.min.js - I'm talking about `-min.js` suffix.
examples:
- https://code.jquery.com/jquery-3.4.1.min.js
- https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js
- https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js

Reply:
applies to the others as well

Reply:
I've wanted to do that but did not find the configuration on the plugin web :D